store/pruning/manager.go

```diff
@@ -74,22 +88,41 @@ func (m *Manager) HandleSnapshotHeight(height int64) {
 		return
 	}
 
+	m.logger.Debug("HandleSnapshotHeight", "height", height)
+
 	m.pruneSnapshotHeightsMx.Lock()
 	defer m.pruneSnapshotHeightsMx.Unlock()
 
-	m.logger.Debug("HandleSnapshotHeight", "height", height)
+	// remove from the in-flight list
+	if pos := slices.Index(m.inflightSnapshotHeights, height); pos != -1 {
```

Comment:
```suggestion
	if position := slices.Index(m.inflightSnapshotHeights, height); pos != -1 {
```

Reply:
👍 I prefer shorter names but I am not opinionated. My rule of thumb is: the shorter the scope is where the var is used, the shorter the name can be. And vise versa of course.